### ISSUE TITLE
Allow to set clientId and ClientKey via gradle properties

### ISSUE THREAD

**Issue by MazyNoc** [cianru/huawei-publish-gradle-plugin#9]
Right now, the client id and key must be in a json file which makes it harder to "hide" the values locally (of course, we could put the file in .gitIgnore or outside the project, but a nicer approach would be to allow it to be populated directly in the `huaweiPublish` tag

example :
```
huaweiPublish {
    instances {
        release {
           // credentialsPath = "$rootDir/app/huawei-credentials.json"
            clientId = project.property("huawei_client_id")            
            clientSecret = project.property("huawei_client_secret")            // secret as from PR #8
            buildFormat = "apk"
            publish = false
        }
    }
}
```
This allows the user to just add these variables into the global gradle.properties and share that file among developers (one file for all projects)

**Comment by MazyNoc** [comment]
There is no reason to remove the current `credentialsPath`, just clarify which ones are taking precedence in the documentation

**Comment by CedrickFlocon** [comment]
https://github.com/cianru/huawei-publish-gradle-plugin/pull/8#discussion_r500802383

**Comment by cosic** [comment]
Released at `v1.2.1`
